backends/audiocd/default/default-audiocd.cpp

```diff
@@ -68,15 +107,13 @@ bool DefaultAudioCDManager::play(int track, int numLoops, int startFrame, int du
 		// Try to load the track from a compressed data file, and if found, use
 		// that. If not found, attempt to start regular Audio CD playback of
 		// the requested track.
-		Common::String trackName[4];
-		trackName[0] = Common::String::format("track%d", track);
-		trackName[1] = Common::String::format("track%02d", track);
-		trackName[2] = Common::String::format("track_%d", track);
-		trackName[3] = Common::String::format("track_%02d", track);
+		Common::Array<Common::String> trackNames;
```

Comment:
Minor remark: you can use `Common::StringArray`.
In practice this doesn't change anything since this is a type for `Array<String>`.